src/service/models.py

```diff
@@ -173,6 +176,14 @@ class Collection(BaseModel):
     matchers: list[Matcher] = Field(
         description="The matchers associated with the collection"
     )
+    default_select: str | None = Field(  # might need to make this a list in future...? not sure
+        **DATA_PRODUCT_ID_FIELD_PROPS | {
+        "description":
+            "The ID of the data product to which non-data product specific selections "
```

Comment:
I'm not sure I understand what this means

Reply:
Does this essentially mean selections are "part" of the genome attribs data product for now?

Reply:
It tells the system which data product is the "primary"  or "default" in terms of where to initially apply the selection. For now, the only choice we have is genome attribs, but this allows setting a different data product in the future.

So when saving a collection, you'd specify which data product the selection should initially be applied to here, and then other data products would calculate their selections from that primary selection.

Reply:
On the other hand, a selection against taxa counts couldn't work that way so it would deal with figuring out the selection itself